<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportMetadataWizard</class>
 <widget class="QWizard" name="ExportMetadataWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="SelectionPage">
   <property name="title">
    <string>Export Metadata</string>
   </property>
   <property name="subTitle">
    <string>This wizard will guide you through the process of exporting metadata stored in Praaline corpus repositories to other formats.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayoutTreeViews">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxRepositoriesCorpora">
        <property name="title">
         <string>Select Repository and Corpora</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QTreeView" name="treeViewRepositoriesCorpora"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelRepository">
           <property name="text">
            <string>Repository:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxRepository"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelCorpora">
           <property name="text">
            <string>Corpora:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBoxCorpusObjectTypes">
        <property name="title">
         <string>Select Corpus Object Types</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTreeView" name="treeViewCorpusObjectTypes"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBoxMetadataAttributes">
        <property name="title">
         <string>Select Metadata Attributes</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTreeView" name="treeViewMetadataAttributes"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayoutTop">
      <item row="1" column="2">
       <widget class="QPushButton" name="commandSelectFileFolder">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QGridLayout" name="gridLayoutExport">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxExportFormat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="editFileFolder"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileFolder">
        <property name="text">
         <string>File / Folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelExportFormat">
        <property name="text">
         <string>Export format:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="ExportingPage">
   <property name="title">
    <string>Export Metadata</string>
   </property>
   <property name="subTitle">
    <string>Click Finish to import files and data according to the choices made during the previous steps.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="texteditMessages"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxExportFormat</tabstop>
  <tabstop>editFileFolder</tabstop>
  <tabstop>commandSelectFileFolder</tabstop>
  <tabstop>comboBoxRepository</tabstop>
  <tabstop>treeViewRepositoriesCorpora</tabstop>
  <tabstop>treeViewCorpusObjectTypes</tabstop>
  <tabstop>treeViewMetadataAttributes</tabstop>
  <tabstop>texteditMessages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
